java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/ErrataHandler.java

```diff
@@ -1060,9 +1022,10 @@ public int cloneAsync(User loggedInUser, String channelLabel,
 
         List<Errata> errataToClone = new ArrayList<Errata>();
         List<Long> errataIds = new ArrayList<Long>();
+        Optional<Org> originalChannelOrg = ofNullable(original).map(c -> c.getOrg());
         //We loop through once, making sure all the errata exist
         for (String advisory : advisoryNames) {
-            Errata toClone = lookupErrata(advisory, errataOrg);
+            Errata toClone = lookupAccessibleErratum(advisory, originalChannelOrg, loggedInUser.getOrg());
```

Comment:
Are we sure this is the only place where we should use the new method?

Reply:
@chiaradiamarcelo I guess the problem in this case is that the new `lookupAccessibleErratum` method throws an exception if nothing is found, but in other cases we just want to return `null`, am I missing something?

Reply:
@ncounter the "old" method also throws the same exception in case nothing is found.

Reply:
Oh, you are right. Only our *new-but-old* `lookupErratumByAdvisoryAndOrg` method doesn't :man_facepalming:

Reply:
1. I only replaced the usages of the `lookupErratumByAdvisoryAndOrg` (even this goes beyond the scope of the bugfix in this PR - I don't want to do the refactoring as a part of the bugfix). I think possible replacing of the  `lookupVendorAndUserErrataByAdvisoryAndOrg` method deserves a separate PR.


1. 
> Oh, you are right. Only our new-but-old lookupErratumByAdvisoryAndOrg method doesn't 

I think it does, because it called `lookupVendorAndUserErrataByAdvisoryAndOrg` internally.